src/document/DocumentAnnotator.ts

```diff
@@ -38,8 +36,8 @@ export default class DocumentAnnotator extends BaseAnnotator {
         return managers;
     }
 
-    getPageNumber(pageEl: HTMLElement): string {
-        return pageEl.dataset.pageNumber || '1';
+    getPageNumber(pageEl: HTMLElement): number {
+        return parseInt(pageEl.dataset.pageNumber || '', 10) || 1;
```

Comment:
may be `parseInt(pageEl.dataset.pageNumber || '1', 10)`

Reply:
Yeah, but the current form protects against `NaN` if pageNumber is a weird string for some reason. It seems minor either way.